services/publicapi/test/send_transaction_test.go

```diff
@@ -112,3 +112,27 @@ func TestSendTransaction_TimesOut(t *testing.T) {
 		require.NotNil(t, result, "Send transaction returned nil instead of object")
 	})
 }
+
+func TestSendTransaction_ReturnImmediately(t *testing.T) {
+	test.WithContext(func(ctx context.Context) {
+		timeout := 100 * time.Second // won't actually wait please don't change
+		harness := newPublicApiHarness(ctx, timeout)
+
+		txb := builders.Transaction().Builder()
+		harness.onAddNewTransaction(func() {})
```

Comment:
suggest cleaning up this line for readability, passing an empty func(){} for no reason ins't nice